index.d.ts

```diff
@@ -0,0 +1,30 @@
+import { Group } from 'ol/layer';
+import BaseLayer from 'ol/layer/Base';
+import Control, { Options as ControlOptions } from 'ol/control/Control';
+import PluggableMap from 'ol/PluggableMap';
+
+type GroupSelectStyle = 'none' | 'children' | 'group';
+
+export interface Options extends ControlOptions {
+    activationMode?: string;
```

Comment:
`activationMode` can be either `'click'` or `'mouseover'`, should we have a union type for this?

Reply:
yeah. this should work.  I'll try to get around to pushing an update today.

`activationMode?: 'click' | 'mouseover;`

Reply:
i pushed a change for this